folium/plugins/beautify_icon.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import (absolute_import, division, print_function)
+
+import json
+
+from branca.element import Figure, JavascriptLink, CssLink, MacroElement
+
+from six import iteritems
+
+from jinja2 import Template
+
+
+class BeautifyIcon(MacroElement):
+    """
+    Create a BeautifyIcon that can be added to a Marker
+    Parameters
+    ----------
+    icon: string
```

Comment:
If you want to make this perfect, you could add the default values to each item. Like in the normal `Icon` class.